pkg/applier/applier_test.go

```diff
@@ -226,46 +248,80 @@ func TestApply(t *testing.T) {
 			events: []event.Event{
 				formPruneSkipEventWithDependency(deploymentID),
 			},
-			multiErr: SkipErrorForResource(
+			expectedError: SkipErrorForResource(
 				errors.New("dependent delete actuation failed: namespace_name_group_kind"),
 				idFrom(deploymentID),
 				actuation.ActuationStrategyDelete),
-			gvks: map[schema.GroupVersionKind]struct{}{
+			expectedGVKs: map[schema.GroupVersionKind]struct{}{
 				kinds.Deployment(): {},
 				testGVK:            {},
 			},
 		},
+		{
+			name: "abandon object",
+			serverObjs: []client.Object{
+				abandonObj,
+			},
+			events: []event.Event{
+				formPruneSkipEventWithDetach(abandonObj),
+			},
+			expectedError: nil,
+			expectedGVKs: map[schema.GroupVersionKind]struct{}{
+				kinds.Deployment(): {},
+				testGVK:            {},
+			},
+			expectedServerObjs: []client.Object{
+				func() client.Object {
+					obj := abandonObj.DeepCopy()
+					obj.SetAnnotations(map[string]string{
+						common.LifecycleDeleteAnnotation: common.PreventDeletion,
+						// all configsync annotations removed
```

Comment:
all configsync annotations **_and labels_** removed

Reply:
fixed & added labels to test